### PR TITLE
By default output only a single file from djxl.

### PR DESCRIPTION
**Pull request by szabadka**
Added separate flags for dumping all extra channels and frames into separate files.

Addresses Issue #2813

### CODE REVIEW

File: tools/djxl_main.cc [hunk 1]
```diff
@@ -149,6 +149,20 @@ struct DecompressArgs {
                            "No output file will be written (for benchmarking)",
                            &disable_output, &SetBooleanTrue, 2);
 
+    cmdline->AddOptionFlag('\0', "output_extra_channels",
+                           "If set, all extra channels will be written either "
+                           "as part of the main output file (e.g. alpha "
+                           "channel in png) or as separate output files with "
+                           "suffix -ecN in their names.",
+                           &output_extra_channels, &SetBooleanTrue, 2);
```
[the remark this answers is not in the document]
Comment (szabadka) [reply tools/djxl_main.cc:157]: It is a no-op if we have only alpha as an extra channel and the output is PNG. That is what I was trying to describe in the help string:  "either as part of the main output (e.g. alpha channel in png)"

[the remark this answers is not in the document]
Comment (mo271) [reply tools/djxl_main.cc:157]: Perhaps add "If not set, the (first) alpha channel will only be written when the output format supports alpha channels and all other extra channels won't be decoded"

[the remark this answers is not in the document]
Comment (mo271) [reply tools/djxl_main.cc:157]: also not clear to me how this interacts with spot colors

[the remark this answers is not in the document]
Comment (szabadka) [reply tools/djxl_main.cc]: > Perhaps add "If not set, the (first) alpha channel will only be written when the output format supports alpha channels and all other extra channels won't be decoded"

Done.

**Review by mo271 — APPROVED** [review]

**Pull request merged**